### PR TITLE
Make it compatible with OpenShift

### PR DESCRIPTION
**Pull request by mfurquim**
- Modify identification from Docker ID to IP
- Increase number of hashes to use on consistent ring, thus making more balanced

### CODE REVIEW

File: promster/main.go [hunk 1]
```diff
@@ -290,30 +287,26 @@ func updatePrometheusTargets(scrapeTargets []string, promNodes []string) error {
...
-	// 	logrus.Warnf("Could not reload Prometheus configuration. err=%s. output=%s", err0, output)
-	// }
 
 	return nil
 }
 
 func keepSelfNodeRegistered(reg *etcdregistry.EtcdRegistry, etcdServiceName string, ttl time.Duration) {
 	node := etcdregistry.Node{}
 	node.Name = getSelfNodeName()
-	logrus.Debugf("Registering Prometheus instance on ETCD registry. service=%; node=%s", etcdServiceName, node)
```
Comment (mfurquim) [inline promster/main.go]: Estava faltando um `s` no `%s` do service.

**Comment by flaviostutz** [comment]
Thanks for sharing, @mfurquim! The hashList() you used to enhance consistent hash does have a real effect? Theoretically I think it shouldn’t. Do you have a test case showing the results?

If you wish, create a separate PR for using local IP instead of hostname as Node ID.

Thanks!

**Comment by mfurquim** [comment]
Will do!

I'll make a test to see whether it does improve or not the consistent hash. I had this idea based on [this](https://youtu.be/zaRkONvyGr8?t=337) video which explains the hash ring and it faults. I have tested the `hashring.New` function, which creates the ring, and it appears to not hash the input. Since we must provide the hash for the function, I also assumed it does not implement multiple hashes for a single note for a better balance, as explained in the video.

**Comment by mfurquim** [comment]
It is done, @flaviostutz. This PR is for making promster compatible with OpenShift. See [this](https://gist.github.com/mfurquim/54ecf6d4dbd3d66c5739c8eb142a1780) gist for testing I've done on hashring use. I'll make a more complete test to verify if multiple hashes makes for better balance.

**Comment by flaviostutz** [comment]
Great work! Thanks for the enhancement!

**Pull request merged**